EMIP/1-1000/EMIP00143.xml

```diff
@@ -238,6 +239,103 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                        </layout>
                      </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                  <handNote xml:id="h1" script="Ethiopic">
+                  <seg type="rubrication">
+                    <ref target="#ms_i1.4 #ms_i1.5"/>
+                  <foreign xml:lang="gez">ኢግዚአብሔር፡</foreign>
+                  <ref target="#ms_i1.1 #ms_i1.2 #ms_i1.3"/>
+                  <foreign xml:lang="gez">ማርያም፡</foreign>
+                  </seg>
+                  <desc>Columetric layout of text: On <locus target="#152r"/> (Ps. 150) the third letter of the first word has been written in red ink; no columetric layout of the tenth biblical canticle.</desc>
+                  </handNote>
+                  </handDesc>
+                  <decoDesc>
+                    <summary>(by the speckled-garment artist)</summary>
+                    <decoNote type="miniature" xml:id="d01"><locus target="#1r"/><desc><ref type="authFile" corresp="AT1031VirginAndChild">, (painted over the top of the first folio of text of the Psalms of David)</ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d02"><locus target="#25r"/><desc><ref type="authFile" corresp="AT1025NativityJesus"></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d03"><locus target="#49r"/><desc><ref type ="authFile" corresp="AT1002Annunciation"/></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d04"><locus target="#73r"/><desc><ref type ="authFile" corresp="AT1062CoronationMary"/></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d05"><locus target="#92v"/><desc><ref type="authFile" corresp="AT1082Angel">, two angels each with a <term key="sword"/></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d06"><locus target="#93r"/><desc><ref type="authFile" corresp="AT1082Angel">, <persName ref="PRS4377Gabriel"/> visits the three young men in the furnace</ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d07"><locus target="#113r"/><desc><ref type="authFile" corresp="AT1066DavidHarp"></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d08"><locus target="#143r"/><desc><ref type="authFile" corresp="AT1038Baptism"></ref> with caption (l. to r.): <q>“How Christ was baptized,” “God the Father,” “Christ,” “Holy Spirit,” “Baptism,” and “Saint John.”</q></desc></decoNote>
```

Comment:
sorry if I missed it before. this <q/> elements should have @xml:lang and eventually only contain the actual text of the legend.

Reply:
also the double quotes can be omitted, as I can add them in the visualization instead